generators/php/lists.js

```diff
@@ -356,6 +356,34 @@ Blockly.PHP['lists_getSublist'] = function(block) {
   return [code, Blockly.PHP.ORDER_FUNCTION_CALL];
 };
 
+Blockly.PHP['lists_sort'] = function(block) {
+  // Block for sorting a list.
+  var listCode = Blockly.PHP.valueToCode(block, 'LIST',
+      Blockly.PHP.ORDER_FUNCTION_CALL) || 'array()';
+  var direction = block.getFieldValue('DIRECTION') === '1' ? 1 : -1;
```

Comment:
At Google we don't use === unless there's something crazy going on (like we care about null vs undefined).  Each of the generators can save a byte here.  Other companies have the opposite policy and avoid == in all cases.